src/database/seeders/20190819153902-user.js

```diff
@@ -11,6 +22,7 @@ module.exports = {
         password: 'jammy11167',
         createdAt: new Date(),
```

Comment:
let remove this a default value is created already for createdAt and    updatedAt

Reply:
Done